paper-toast.html

```diff
@@ -48,6 +48,7 @@
 ----------------|-------------|----------
 `--paper-toast-background-color` | The paper-toast background-color | `#323232`
 `--paper-toast-color` | The paper-toast color | `#f1f1f1`
+`--paper-font-common-base` | Mixin applied to the toast for setting the font, defined in `paper-styles/typography.html` | `{}`
```

Comment:
`paper-styles` is already a dependency of this element (not a devDependency), so what if we have something like
>  Mixin applied to the toast for setting the font. Roboto is the default font used (defined in paper-styles/typography.html); you can define your own font style by overriding this mixin.